src/components/Tooltip/Tooltip.js

```diff
@@ -142,7 +141,7 @@ export default class Tooltip extends Component {
   };
 
   state = {
-    open: this.props.open,
+    open: this.props.open === undefined ? false : this.props.open,
```

Comment:
Would we anticipate non-boolean and non-null/undefined values for `state.open`? If not, wondering if we can do `Boolean(this.props.open)`.

Reply:
I will make the code change. thanks for pointing our